### PR TITLE
fix: reanimated crashes fix pt2

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [] 

### Description

Fixed the reanimated crashes weirdness

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- reanimated crashes - gkartalis

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Scenes/SellWithArtsy/Components/Testimonials.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Flex, Spacer, Text, useColor } from "@artsy/palette-mobile"
 import { useSWALandingPageData } from "app/Scenes/SellWithArtsy/utils/useSWALandingPageData"
 import { isPad } from "app/utils/hardware"
+import { useScreenDimensions } from "app/utils/hooks"
```
Comment (gkartalis) [inline src/app/Scenes/SellWithArtsy/Components/Testimonials.tsx:4]: ignore, linting change

File: src/app/Scenes/SellWithArtsy/Components/Testimonials.tsx [hunk 2]
```diff
@@ -110,7 +110,11 @@ const Dot: React.FC<{ currentIndex: number; index: number }> = ({ currentIndex,
   const inactiveColor = color("black15")
 
   const isActive = useDerivedValue(() => {
-    return withTiming(currentIndex === index ? 1 : 0)
+    if (currentIndex === index) {
+      return withTiming(1)
+    }
+
+    return withTiming(0)
```
Comment (gkartalis) [inline src/app/Scenes/SellWithArtsy/Components/Testimonials.tsx:117]: apparently reanimated hooks don't like implicit returns

**Comment by gkartalis** [comment]
First I will merge the updated palette with this fix and when this is merged no more issues hopefully 🙏

**Review by olerichter00 — APPROVED** [review]

File: src/app/Scenes/SellWithArtsy/Components/Testimonials.tsx (hunk 2 above)
Comment (pvinis) [reply src/app/Scenes/SellWithArtsy/Components/Testimonials.tsx:117]: 🤦

**Pull request merged**